app/javascript/application.js

```diff
@@ -3,11 +3,14 @@ import "@hotwired/turbo-rails"
 import "controllers"
 import bootstrap from "bootstrap"
 window.bootstrap = bootstrap // Required for Blacklight 7 so it can manage the modals
-import "blacklight/blacklight"
+import "@github/auto-complete-element"
+import "blacklight"
```

Comment:
Does this change imply you should remove https://github.com/sul-dlss/arclight-demo/pull/346/files#diff-f8ec49173826da81af1372f887fe55932973f785fc1a1d5555cb173bd8135d63R11?

Reply:
Maybe so. This change along with line 6 fixed the auto-complete UI gremlin that also appeared in vt-arclight a while back: https://github.com/sul-dlss/vt-arclight/issues/538.